packages/cli/src/ui/App.tsx

```diff
@@ -88,6 +88,13 @@ import { PrivacyNotice } from './privacy/PrivacyNotice.js';
 
 const CTRL_EXIT_PROMPT_DURATION_MS = 1000;
 
+interface KeystrokeHandler {
```

Comment:
this is worth moving to its own file rather than keeping in app.tsx
I'd like to reuse in InputPrompt and text-buffer.ts as well.
In addition, I'm hoping this can also be the basis to in the future configure keyboard shortcuts via a file and surface the up to date keyboard shortcuts in `/help`.

Reply:
that's a great idea! added to parent issue